test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -1891,11 +1891,11 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 				if clusterAutoscalerEngaged {
 					cpuTarget = 50
 					for _, profile := range eng.ExpandedDefinition.Properties.AgentPoolProfiles {
-						maxPods, _ := strconv.Atoi(profile.KubernetesConfig.KubeletConfig["--max-pods"])
-						var n []node.Node
-						n, err = node.GetByRegexWithRetry(fmt.Sprintf("^k8s-%s", profile.Name), 3*time.Minute, cfg.Timeout)
```

Comment:
So now we're just using `profile.Count` instead of actually fetching nodes to find the `maxTotalPods`. Is this because of Windows, and was there value in making the API call to get the nodes?

Reply:
Right, it's because the prior regex would fail (Windows nodes have a different name).

The purpose of this enumeration is to set an appropriate number of pod replicas that exceeds the pods-per nodes setting across all nodes (so that we induce quick scale up by creating pods that are unable to be scheduled). Doing the node count by calling the k8s api at runtime is more strictly accurate, but with Windows nodes in the mix, got more complicated: basically correlating the nodes in the cluster with a particular pool is trickier. All that being said, the theoretical benefits of doing this over simply taking the `Count` value in the api model seemed to be not worth wrangling this complexity.